docker-compose.backend.yml

```diff
@@ -69,6 +69,13 @@ services:
     restart: unless-stopped
     ports:
       - 8005:8005
+
+  collaboration-service:
+    container_name: collaboration-service
+    build: ./collaboration-service
+    restart: unless-stopped
+    ports:
+      - 8003:8003
```

Comment:
Minor issue, put this between the services using port 8002 and 8004 for easier readability